README.md

```diff
@@ -5,6 +5,13 @@
 
 When the CRDs are present in the cluster, the operator can either be run outside or inside the cluster.
 
+## Prerequisites/Dependencies
+### cert-manager
+We use admission webhooks for managing the CRDs. Therefore, there's a need for TLS certificates and keys management.
```

Comment:
Would be good to add with which podman version we are aligned. For each new podman update we would need to make sure the validation is still correct.

Reply:
we have not yet pinned a version. I think that podman is a key component so we should document it well somewhere else.

Reply:
I added podman. please let me know if it's ok.

Reply:
@pkliczewski please review my fixes